lib/tests/streamlit/color_picker_test.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2018-2020 Streamlit Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""color_picker unit test."""
+
+from tests import testutil
+import streamlit as st
+from parameterized import parameterized
+
+
+class ColorPickerTest(testutil.DeltaGeneratorTestCase):
+
+    def test_just_label(self):
+        """Test that it can be called with no value."""
+        st.color_picker("the label")
+
+        c = self.get_delta_from_queue().new_element.color_picker
+        self.assertEqual(c.label, "the label")
+        self.assertEqual(c.default, "#000000")
+
+    @parameterized.expand(
+        [("#333333", "#333333"), (None, "#000000")]
+    )
+    def test_value_types(self, arg_value, proto_value):
```

Comment:
Can we test also `#333`?

Reply:
I'm not sure if `None` should be accepted as input. 

Perhaps the API should either accept only strings, or accept anything and cast it to a string. I pinged @tvst about this in slack.